packages/lodestar/src/metrics/metrics.ts

```diff
@@ -18,14 +19,15 @@ export function createMetrics(
   opts: IMetricsOptions,
   config: IChainForkConfig,
   anchorState: BeaconStateAllForks,
+  logger: ILogger,
```

Comment:
Do we need to be logging the validator monitor / metrics?

Reply:
yeah at the first look it's a little bit strange to add logger to metric, but we need it to log attestation journey since `ValidatorMonitor` is the only place that manage our connected validators